src/context/index.ts

```diff
@@ -0,0 +1,13 @@
+import { createContext } from 'react';
+
+interface ThemeContextType {
+  theme: string;
+  setTheme: React.Dispatch<React.SetStateAction<string>>;
```

Comment:
Referenced UMD global variable
```suggestion
  setTheme: Dispatch<SetStateAction<string>>;
```

Reply:
Thanks